### DISCUSSION TITLE
Error when i try to login

### DISCUSSION THREAD

**Discussion by enesok** [znck/grammarly#344] [Q&A] [answered]
Hey, im getting this error when i try to login.
Request $/handleOAuthCallbackUri failed with message: Minified error #8:  See https://developer.grammarly.com/docs/error-explainer?code=8&args=vscode%3A%2F%2Fznck.grammarly%2Fauth%2Fcallback%3FwindowId%3D3&trace=Error%0A++++at+https%3A%2F%2Fjs.grammarly.com%2Fgrammarly-sdk%401.7%3A17%3A60282%0A++++at+Generator.next+%28%3Canonymous%3E%29%0A++++at+bt+%28https%3A%2F%2Fjs.grammarly.com%2Fgrammarly-sdk%401.7%3A17%3A26642%29%0A++++at+o+%28https%3A%2F%2Fjs.grammarly.com%2Fgrammarly-sdk%401.7%3A17%3A26846%29%0A++++at+https%3A%2F%2Fjs.grammarly.com%2Fgrammarly-sdk%401.7%3A17%3A26907%0A++++at+new+Promise+%28%3Canonymous%3E%29%0A++++at+https%3A%2F%2Fjs.grammarly.com%2Fgrammarly-sdk%401.7%3A17%3A26786%0A++++at+https%3A%2F%2Fjs.grammarly.com%2Fgrammarly-sdk%401.7%3A17%3A90543%0A++++at+t.SDK.handleOAuthCallback+%28https%3A%2F%2Fjs.grammarly.com%2Fgrammarly-sdk%401.7%3A17%3A90565%29%0A++++at+%2FUsers%2Fenesoeksuez%2F.vscode%2Fextensions%2Fznck.grammarly-0.22.1%2Fdist%2Fserver%2Findex.node.js%3A77%3A1019 for details.

**Comment by vusal-iom** [comment]
Same here

**Reply by lars-nagel** [reply]
Same for me

**Comment by shesmer-cn** [comment]
Same

**Comment by IvanKostyuk** [comment]
I switched to pre-release, the error has gone, but it says:
Your Grammarly account is not used for this file.
Connection status: idle.

**Comment by songypp** [comment]
same here

**Comment by enesok** [comment, marked as answer]
Switch to 0.18.0 or 0.18.1 and than login. Afterwards you can switch back to 0.22. After some days im getting kicked out. Then you have to repeat this procedure. You can also just stay at 0.18.0.

**Reply by songypp** [reply]
works for me! Thanks

**Reply by TheoKondak** [reply]
Worked for me too. Let's see if I get kicked out

**Reply by lars-nagel** [reply]
Works!

**Comment by lighterseller** [comment]
Yep, thanks!

**Comment by andrewehe** [comment]
Same here. Downgrading to 18.0 worked for me.

**Comment by loicmathieu** [comment]
Same here, Downgrading worked.

**Comment by chienhsiang-hung** [comment]
downgraded but still stuck at the same error

**Reply by lavrovpy** [reply]
Do you use VS Code Insiders? If yes, try to install the extension in the stable version of VS Code.
